### PR TITLE
Homework/task 3

### PR DESCRIPTION
**Pull request by AlexanderPhilippov**

### CODE REVIEW

File: src/Components/ErrorBoundary/index.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                        <>{error?.stack}</>
+                    </div>
+                </div>
+            </div>
+        </div>
+    )
+}
+
+
+class ErrorBoundary extends React.Component<ErrorBoundaryProps, ErrorBoundaryState> {
```
Comment (mmarkelov) [inline src/Components/ErrorBoundary/index.tsx:23]: Not an error, but you can use `React.PropsWithChildren<P>`:
```js
type ComponentWithChildProps = React.PropsWithChildren<{}>;
```

File: src/Components/Header/index.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    return (
+        <div className="header">
+            <img className="header__image" src={image} />
+            <div className="header__text-logo">
+                <span>netflix</span>roulette
+            </div>
+            <input className="header__add-movie_button" type="button" value="+ add movie" />
+            <div className="header__search">
+                <label htmlFor="searchMovieInput">Find your movie</label>
+                <input id="searchMovieInput" type="text" placeholder="What do you want to watch?" />
```
Comment (mmarkelov) [inline src/Components/Header/index.tsx:15]: For input default type is **text**, so you can just skip it

File: src/Components/Header/styles.scss [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            font-size: large;
+            &:hover {
+                background-color: #e63544;
+            }
+        }
+
+        input[type="text"]{
+            width: 80%;
+            margin-right: 10px;
+            background-color: #323232CC;
```
Comment (mmarkelov) [inline src/Components/Header/styles.scss:80]: is it valid color?

File: src/Components/Movie/styles.scss [hunk 4]
```diff
@@ -0,0 +1,45 @@
+.movie-card {
+    margin-bottom: 50px;
+    flex: 0 1 30%;
+    display: flex;
+    flex-wrap: wrap;
+    color: #ffff;
```
Comment (mmarkelov) [inline src/Components/Movie/styles.scss:6]: shouldn't it be `#fff`?

File: src/Components/MovieList/index.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+import { MoviesResponseModel } from './models'
+import MoviesMockData from './movies.json'
+import './styles.scss'
+import { MoviesHeader } from './Header'
+
+const MovieList: React.FC = () => {
+    return (
+        <div className="movie-list">
+            <MoviesHeader total={MoviesMockData.totalAmount}/>
+            {(MoviesMockData as MoviesResponseModel)?.data.map(x => (
```
Comment (mmarkelov) [inline src/Components/MovieList/index.tsx:12]: can it be `movie` instead of `x`?

File: src/styles.scss [hunk 6]
```diff
@@ -1,4 +1,6 @@
 body{
     margin: 0;
     padding: 0;
+    background-color: #232323;
+    ;
```
Comment (mmarkelov) [inline src/styles.scss:5]: do you need this?

File: src/Components/ErrorBoundary/index.tsx (hunk 1 above)
Comment (AlexanderPhilippov) [reply src/Components/ErrorBoundary/index.tsx:23]: Got it, thank you

File: src/Components/Header/index.tsx (hunk 2 above)
Comment (AlexanderPhilippov) [reply src/Components/Header/index.tsx:15]: Ok, will know now, thank you :)

File: src/styles.scss (hunk 6 above)
Comment (AlexanderPhilippov) [reply src/styles.scss:5]: Yeah... my fault, I have not still configure StyleLint.

File: src/Components/MovieList/index.tsx (hunk 5 above)
Comment (AlexanderPhilippov) [reply src/Components/MovieList/index.tsx:12]: sure will change

File: src/Components/Movie/styles.scss (hunk 4 above)
Comment (AlexanderPhilippov) [reply src/Components/Movie/styles.scss:6]: Yeah... you right! I have copied this value from scrapbook
Will change

File: src/Components/Header/styles.scss (hunk 3 above)
Comment (AlexanderPhilippov) [reply src/Components/Header/styles.scss:80]: yes, this value from figma's scrapbook

**Pull request merged**